File: Julia/Problem009.jl
# Parameters:
sum_ = 1000
################

for a in 1 : sum_ - 2
	for b in 1 : sum_ - a - 1
		c = sum_ - a - b
		if a^2 + b^2 == c^2
			println(a * b * c)
			return
		end
	end
end

File: C++/Problem009.cpp
#include <iostream>


int main() {
	// Parameters:
	int S {1000};
	///////////////

	for(int a = 1; a <= S - 2; a++) {
		for(int b = 1; b <= S - a - 1; b++) {
			int c = S - a - b;

			if(a * a + b * b == c * c) {
				std::cout << a * b * c << std::endl;
				return 0;
			}
		}
	}

	return 0;
}